### DISCUSSION TITLE
How easy/hard will it be to install Athens RTC on a server?

### DISCUSSION THREAD

**Discussion by latvia234** [athensresearch/athens#1577] [General]
When reading the blog (https://athensresearch.ghost.io/season-2/) you can see:
> How do I run and set it up? We will write documentation for all of this.

You can also see:

> The goal for Athens is to serve everyone.

How easy or hard will it be to install Athens RTC for everyone? I'm just a simple user with no experience in coding. I can install a WordPress installation but that's about it. ;-)

Could you make installing Athens RTC as easy as possible for people who have no experience with coding?

It would be great if I could install Athens RTC to show it to my colleagues.

**Comment by latvia234** [comment]
Related: https://github.com/athensresearch/athens/discussions/868 https://github.com/athensresearch/athens/issues/873 https://github.com/athensresearch/athens/issues/842 
Found via search: https://github.com/athensresearch/athens/issues?q=encryption
The answer is here: https://discord.com/channels/708122962422792194/800442937602342922/880414446596534292

**Comment by solarkraft** [comment]
In the current version it's super easy. The server component is shipped as a java archive, so you just do `java -jar athens-lan-party-standalone.jar` on the server (with some extra systemd/docker stuff if you want it to be persistent). Since you mention Wordpress it may be notable that you can't do this in a webspace where you just have access to the file structure - you will need full access to the server ("root", "vserver", "vps") to do this (more expensive to rent, but not much more).
